Makefile

```diff
@@ -183,6 +183,10 @@ run-dev-node:
 run-dev-node-release:
 	@ ./scripts/gear.sh run node --release -- --dev
 
+.PHONY: run-dev-node-release-log
+run-dev-node-release-log:
```

Comment:
Didn't we agree to always put logs in dev mode and remove separated make command?

Reply:
Misunderstanding 😄 fixed it.